lib/src/matchers/jest_dom/is_disabled.dart

```diff
@@ -90,7 +89,7 @@ class _IsDisabled extends Matcher {
     if (!isElementThatCanBeDisabled(item, matchState)) return false;
     final formAncestor = (item as Element).closest('form');
     if (formAncestor != null && (formAncestor as FormElement).hasAttribute('disabled')) return true;
-    return item.disabled as bool;
+    return (item as ButtonElement).disabled;
```

Comment:
This is causing the remaining test failures - we need to figure out a better way to get the `disabled` attribute off an `Element` when only certain sub-types have the attribute

Reply:
Hmm some ideas that come to mind:
- using a dynamic dispatch, casting it to `dynamic` explicitly if necessary: `return (item as dynamic).disabled as bool;`
    - It looks like that might have been what was happening before; does this no longer work?
- potentially sketchier ideas:
    - using CSS selector: `return (item as Element).matches(':disabled');`
    - using custom JS interop: `return getProperty(item, 'disabled') as bool;`

Reply:
Ohh yep the dynamic dispatch worked great! Yeah I think that's what it was doing before, but now it's inferring that `item` is an `Element` which doesn't have `disabled`. Thank you!

Reply:
Nice! Mm that makes sense. Np!